src/app/src/components/PerCapitaMap/SpendingTooltip.tsx

```diff
@@ -117,9 +117,18 @@ function SpendingBars({
 function SpendingBar({ cat, percent }: { cat: Category; percent: number }) {
     return (
         <>
-            <Text>{cat.toString()}:</Text>
-            <Text>{percent}%</Text>
-            <Progress mb={2} colorScheme='tooltip' size='lg' value={percent} />
+            <Text mb={1}>{cat.toString()}:</Text>
+            <HStack spacing={2} mb={2}>
+                <Box height={4} width='140px'>
```

Comment:
This does help keep the text from covering/shortening the bars, but does also block them all to the right side instead of following the right edges of the bars. If this is desired, it can go, but I do think it's much easier and quicker to read if the numbers follow the bars.

![image](https://user-images.githubusercontent.com/64238570/231280512-130de693-2b70-4cce-b37d-a7f1c83628ea.png)

Reply:
Oh duh, I should just give the explicit width to the HStack. Thanks.